Zend/Optimizer/zend_func_infos.h

```diff
@@ -316,31 +316,6 @@ static const func_info_t func_infos[] = {
 	FN("oci_get_implicit_resultset", MAY_BE_RESOURCE|MAY_BE_FALSE),
 	FN("oci_password_change", MAY_BE_RESOURCE|MAY_BE_BOOL),
 	FN("oci_new_cursor", MAY_BE_RESOURCE|MAY_BE_FALSE),
-	FN("odbc_prepare", MAY_BE_RESOURCE|MAY_BE_FALSE),
```

Comment:
As far as I know, these always return a new instance, thus they can be annotated as `@refcount 1`, cannot they?